src/prometheus-net.DotNetRuntime/EventListening/EventParserTypes.cs

```diff
@@ -87,8 +87,16 @@ internal static IEnumerable<Type> GetEventParsers(Assembly fromAssembly)
         internal static Lazy<Version> CurrentRuntimeVerison = new Lazy<Version>(() =>
         {
             var split = RuntimeInformation.FrameworkDescription.Split(' ');
```

Comment:
Can't you use `Environment.Version` here?

Reply:
I absolutely should have, thanks for the comment